telink: zephyr versions compatibility changes

s07641069 opened the pull request:
- use common SoCs names for different Zephyr versions
- update Zephyr SDK version
- telink.yaml: lzma: Enable LZMA Optimize for size by default

andriy-bilynskyy commented:
The following west manifest projects have been modified in this Pull Request:

| Name | Old Revision | New Revision | Diff |
| ---- | ------------ | ------------ |------|
| lzma | https://github.com/telink-semi/xz/commit/dcf34d7ced3a7fffb4ad4862881e0e92c62287d0 | https://github.com/telink-semi/xz/commit/053c1629f452b43339e7cd5c66fc89a143f4827a (telink-zephyr) | [telink-semi/xz@dcf34d7c..053c1629](https://github.com/telink-semi/xz/compare/dcf34d7ced3a7fffb4ad4862881e0e92c62287d0..053c1629f452b43339e7cd5c66fc89a143f4827a) |

*Note: This message is automatically posted and updated by the Manifest GitHub Action.*

andriy-bilynskyy commented:
Here is just board renaming and formatting (CI still shows formatting issues).
SOC_RISCV_TELINK_B9X is already provided by develop branch so this change is not required.
Board renaming seems fine

s07641069 committed:
telink: zephyr versions compatibility changes

- use common SoCs names for different Zephyr versions
- update Zephyr SDK version
- telink.yaml: lzma: Enable LZMA Optimize for size by default

Signed-off-by: Alex Tsitsiura <s07641069@gmail.com>
